src/store/dao/actions.js

```diff
@@ -0,0 +1,172 @@
+import * as Types from './types'
+import { getProviderGasPrice } from '../../utils/web3'
+import { payloadToDispatch, errorToDispatch } from '../helpers'
+import { getDaoContract } from '../../utils/web3Dao'
+
+export const daoLoading = () => ({
+  type: Types.DAO_LOADING,
+})
+
+// --------------------------------------- GENERAL DAO HELPERS (DO TESTS) ---------------------------------------
+
+// Returns the amount of members with LP tokens locked in the DAO
```

Comment:
In my case I would use annotations instead of comment for describing a functions for example

```
/**
 * Returns the amount of members with LP tokens locked in the DAO
 *
 * @return {Object}
 */
```
With annotations the IDE shows tooltips with the description that you put, also you can generate docs with Jdocs

anyway this is only a suggestion

Reply:
thanks, good point about it showing up in tooltips, main point of doing this is so that the more techie types can look through the code and have a good idea about what each function does and learn from it